### ISSUE TITLE
installation failure

### ISSUE THREAD

**Issue by colinhunt** [CoreSecurity/pcapy#25]
Hello, I'm trying to install pcapy on Ubuntu 16.04, but it fails with
```
$ sudo python3 setup.py install
running install
Checking .pth file support in /usr/local/lib/python3.5/dist-packages/
/usr/bin/python3 -E -c pass
TEST PASSED: /usr/local/lib/python3.5/dist-packages/ appears to support .pth files
running bdist_egg
running egg_info
writing top-level names to pcapy.egg-info/top_level.txt
writing dependency_links to pcapy.egg-info/dependency_links.txt
writing pcapy.egg-info/PKG-INFO
reading manifest file 'pcapy.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'pcapy.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
running build_ext
building 'pcapy' extension
x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.5m -c pcapdumper.cc -o build/temp.linux-x86_64-3.5/pcapdumper.o
cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
pcapdumper.cc:11:18: fatal error: pcap.h: No such file or directory
compilation terminated.
error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```
I have `tcpdump` installed. I'm trying to install this for a client, so I'm not very familiar with any of this yet... 

Looks like I do have `pcap.h` in a number of places though:
```
$ locate pcap.h
/usr/src/linux-headers-4.4.0-72/include/linux/mfd/ezx-pcap.h
/usr/src/linux-headers-4.4.0-72-generic/include/config/ezx/pcap.h
/usr/src/linux-headers-4.4.0-72-generic/include/config/input/pcap.h
/usr/src/linux-headers-4.4.0-72-generic/include/config/regulator/pcap.h
/usr/src/linux-headers-4.4.0-72-generic/include/config/rtc/drv/pcap.h
/usr/src/linux-headers-4.4.0-72-generic/include/config/touchscreen/pcap.h
/usr/src/linux-headers-4.4.0-75/include/linux/mfd/ezx-pcap.h
/usr/src/linux-headers-4.4.0-75-generic/include/config/ezx/pcap.h
/usr/src/linux-headers-4.4.0-75-generic/include/config/input/pcap.h
/usr/src/linux-headers-4.4.0-75-generic/include/config/regulator/pcap.h
/usr/src/linux-headers-4.4.0-75-generic/include/config/rtc/drv/pcap.h
/usr/src/linux-headers-4.4.0-75-generic/include/config/touchscreen/pcap.h
/usr/src/linux-headers-4.4.0-78/include/linux/mfd/ezx-pcap.h
/usr/src/linux-headers-4.4.0-78-generic/include/config/ezx/pcap.h
/usr/src/linux-headers-4.4.0-78-generic/include/config/input/pcap.h
/usr/src/linux-headers-4.4.0-78-generic/include/config/regulator/pcap.h
/usr/src/linux-headers-4.4.0-78-generic/include/config/rtc/drv/pcap.h
/usr/src/linux-headers-4.4.0-78-generic/include/config/touchscreen/pcap.h
```
But not in any `lib` folder. 
Any ideas?

**Comment by colinhunt** [comment]
Sorry, I decided to try rtfm and saw that libpcap is the dependency, not tcpdump.

**Comment by MeghaMaheshwari** [comment]
Doing this solved my problem on Ubuntu 16.0

sudo apt-get install libpcap-dev libpq-dev
sudo pip install pcapy
